utils/utils.py

```diff
@@ -207,6 +221,10 @@ def dataset_loading_and_splitting(
             distributed_data_parallelism=distributed_data_parallelism,
         )
     else:
+        raise ValueError(
+            "Not ready yet. Should re-normalize mixed datasets based on joint min_max",
```

Comment:
"Chosen dataset option not yet supported", then maybe a `FIXME` comment about the needed re-normalizations

Reply:
Fixed in 5948ba3